src/sysdiagnose/analysers/timesketch.py

```diff
@@ -17,6 +17,9 @@
 from sysdiagnose.parsers.crashlogs import CrashLogsParser
 from collections.abc import Generator
 from sysdiagnose.utils.base import BaseAnalyserInterface
+import logging
```

Comment:
to replace by the new from sysdiagnose.utils.base import logger?

Reply:
Fixed in [38a0ada](https://github.com/EC-DIGIT-CSIRC/sysdiagnose/pull/106/commits/38a0adaa51877b42414ef819a2ddb46aa366dc68)